packages/common/src/web3IdHelpers.ts

```diff
@@ -21,3 +21,29 @@ export function verifyWeb3IdCredentialSignature(
     // Use json-bigint stringify to ensure we can handle bigints
     return wasm.verifyWeb3IdCredentialSignature(stringify(input));
 }
+
+/**
+ * Compares a and b as field elements.
+ * if a < b then compareStringAttributes(a,b) = -1;
+ * if a == b then compareStringAttributes(a,b) = 0;
+ * if a > b then compareStringAttributes(a,b) = 1;
+ */
+export function compareStringAttributes(a: string, b: string): number {
+    return wasm.compareStringAttributes(a, b);
+}
+
+/**
+ * Given a string attribute value and a range [lower, upper[, return whether value is in the range, when converted into field elements.
+ */
+export function isStringAttributeInRange(
```

Comment:
Make unit tests for the edge cases where `value === lower` and `value === upper`.

Reply:
Added these.
What would we expect if `value === lower === upper`? :thinking:

Reply:
That is invalid input as it would result in an empty interval as `[a, a) = ∅`.